dpdk-metrics-agent/README.md

```diff
@@ -0,0 +1,35 @@
+# DPDK Telemetry Agent
+
+The DPDK telemetry agent is designed to collect the DPDK traffic metrics where the kernel-based network metrics collecor cannot be applied.
+
+## DPDK telemetry API
+
+By default, every DPDK application creates a socket with the path `<runtime_directory>/dpdk_telemetry.<version>`, unless it is disabled during the application's build process. For more details, interested users are encouraged to refer to: https://doc.dpdk.org/guides/howto/telemetry.html.
+
+This agent uses this socket to aquire DPDK traffic metrics from DPDK applications. Through this socket, the agent can monitor a DPDK application's traffic metrics without the need for instrumenting the application.
+
+## Backend support
+
+* Prometheus
+* OpenTelemetry Collector
+
+## Agent command line options
+
+* --sock-prefix, DPDK telemetry socket directory
+* --port, prometheus scaping port
```

Comment:
Should be "scraping port"

Reply:
Good catch. Have a meeting setup to go over it with demo.